core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -733,13 +733,9 @@ static void map_page_memarea_in_pgdirs(const struct tee_mmap_region *mm,
 	attr = mattr_to_desc(1, mm->attr | TEE_MATTR_TABLE);
```

Comment:
To insure only 1 pgdir is field, add `assert(idx == ((mm->va + mm->size) >> SECTION_SHIFT));`

Reply:
@etienne-lms easier to read as `assert(mm->size < SECTION_SIZE);`?

Reply:
no, if `va` is not section aligned, it could spread over the next section(s).

Reply:
Ah yes, obviously. -ENOCOFFEE ;-)